demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -338,14 +338,20 @@ def format_converted_pack(self) -> None:
             f"Executing 'format' on the restructured contribution zip new/modified files at {self.pack_dir_path}"
         )
         from_version = "6.0.0" if self.create_new else ""
-        format_manager(
-            from_version=from_version,
-            no_validate=True,
-            update_docker=True,
-            assume_answer=True,
-            include_untracked=False,
-            interactive=False,
-        )
+        try:
+            format_manager(
+                from_version=from_version,
+                no_validate=True,
+                update_docker=True,
+                assume_answer=True,
+                include_untracked=False,
+                interactive=False,
+            )
+        except Exception as e:
+            if str(e).strip() == "0":
+                logger.info(f"The exit code is {str(e)}")
```

Comment:
Is there other way of getting the exit code? What attributes does e has?

Reply:
From my checks, e is the exit code number without any additional attributes.